kong/tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist =
+  kong
+  flake8
+
+[testenv]
+platform = linux|darwin|win32
+passenv =
+    DOCKER*
+    COMPOSE*
+
+[testenv:kong]
+deps =
+    ../datadog_checks_base
+    -rrequirements-dev.txt
+setenv = KONG_VERSION=0.13.0
+commands =
+    pip install --require-hashes -r requirements.txt
+    pytest -v -r apP --log-level debug
```

Comment:
When everything is passing let's just do `pytest -v`.

Reply:
yea